addons/website_form/views/website_form_templates.xml

```diff
@@ -1,140 +1,184 @@
-                        </form>
+                            <div class="col-lg-4 mt-4 mt-lg-0">
+                                <address itemscope="itemscope" itemtype="http://schema.org/Organization">
+                                    <p class="p-0 m-0">My Company</p>
+                                    <ul class="list-unstyled mb-0 pl-2">
+                                        <li><i class="fa fa-map-marker fa-fw mr-2" /><span class="o_force_ltr">Chaussée de Namur 40</span></li>
+                                        <li><i class="fa fa-phone fa-fw mr-2" /><span class="o_force_ltr">+ 32 81 81 37 00</span></li>
+                                        <li><i class="fa fa-1x fa-fw fa-envelope mr-2" /><span>info@yourcompany.example.com</span></li>
+                                    </ul>
+                                </address>
+                            </div>
+                        </div>
                     </div>
                 </section>
+                <div class="oe_structure mt-2" />
             </div>
-        </xpath>
+        </t>
     </template>
 
     <record id="contactus_thanks" model="website.page">
         <field name="name">Thanks (Contact us)</field>
         <field name="type">qweb</field>
         <field name="url">/contactus-thank-you</field>
-        <field name="website_indexed" eval="False"/>
+        <field name="website_indexed" eval="False" />
         <field name="is_published">True</field>
         <field name="key">website_form.contactus_thanks</field>
         <field name="arch" type="xml">
             <t name="Thanks (Contact us)" t-name="website_form.contactus_thanks">
                 <t t-call="website.layout">
-                  <div id="wrap">
-                    <div class="oe_structure" id="oe_structure_website_form_contact_us_thanks_1"/>
-                    <div class="container mt-4">
-                        <div class="row">
-                            <div class="col-lg-7 col-xl-6 mr-lg-auto oe_structure">
-                                <section class="pt40 s_text_block pb40 o_colored_level o_cc o_cc1" data-snippet="s_text_block" data-name="Text">
-                                    <div class="container">
-                                        <span class="d-block fa fa-4x fa-thumbs-up mx-auto rounded-circle bg-primary"/><br/>
-                                        <h1 class="text-center">Thank You!</h1>
-                                        <div class="pb16 pt16 s_hr" data-snippet="s_hr" data-name="Separator">
-                                            <hr class="mx-auto border-top w-50 border-dark text-center"/>
+                    <div id="wrap">
+                        <div class="oe_structure" id="oe_structure_website_form_contact_us_thanks_1" />
+                        <div class="container mt-4">
+                            <div class="row">
+                                <div class="col-lg-7 col-xl-6 mr-lg-auto oe_structure">
```

Comment:
While we are at it, I don't think there is a reason for the "thank you" page to not be fully static too, confirm that with @sbuhl. But in that case, there won't be two `oe_structure` around the page content and the customization of those in current user database would have to be migrated.

Actually I am seeing your upgrade script, I think it would need to be more complicated than just removing the old views. I need to discuss that with @JKE-be. IMO the minimum should be the disable the new contactus page in old databases and keeping the old ones (as user data). The ideal would be to migrate existing user pages (and their customization) inside the new page structure ("easy" for the empty oe_structure edition, not easy/impossible for their automatic address). To investigate and discuss.